src/app/loginComponents/accounts/external/accounts.component.scss

```diff
@@ -0,0 +1,45 @@
+@use '~@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.account-logo {
+  width: 3.5em;
+  margin-right: 1em;
+}
+
+.token-form-field {
+  width: 300px;
+  font-size: 12px;
+}
+
+.color {
+  color: mat.get-color-from-palette($dockstore-app-accent-1, darker);
+}
+
+.mat-badge-medium.mat-badge-above .mat-badge-content {
+  top: 0px;
+  left: 40px;
+  background: #26bfa5;
+}
+
+.card-style {
+  height: 100%;
+}
+
+.button-size {
+  transform-origin: 100% 50%;
```

Comment:
what's so special about this button that it needs to have a different size than all buttons on the entire site?

Reply:
I made the Unlink/Link button smaller since it seemed smaller based on the mockup, also thought it would look nicer overall. Open to opinions with keeping default size or making it smaller. 

Here is how it would look without the change in size:
<img width="1229" alt="Screen Shot 2021-08-16 at 6 02 01 PM" src="https://user-images.githubusercontent.com/65895867/129636547-95e72c23-7d9b-4a52-8844-ecdf88ca1521.png">